widget/embedded/src/store/quote.ts

```diff
@@ -289,3 +296,55 @@ export const useQuoteStore = createSelectors(
     }))
   )
 );
+
+useQuoteStore.subscribe((selectedState, previousSelectedState) => {
```

Comment:
The subscribe method returns an unsubscribe function. You can use this in the `useBootstrap` hook to unsubscribe from the store and avoid unexpected results.

Reply:
Tnx, Done.